ceph-pull-requests/config/definitions/ceph-pull-requests.yml

```diff
@@ -91,7 +91,10 @@
                 - shell:
                     !include-raw:
                       - ../../build/kill-tests
-
+      - xunit:
+          types:
+            - ctest:
+                pattern: "build/Testing/**/Test.xml"
```

Comment:
junit has a new interesting feature: it [integrates with Github checks](https://plugins.jenkins.io/junit/), but unfortunately I don't think the XML generated by [ctest is XML JUnit compliant](https://gitlab.kitware.com/cmake/cmake/-/issues/18654).

Reply:
yeah, i like Github Check APIs --  it integrates better with github, and one less click =)

but, probably we can use some XSL for translating the ctest XML to JUnit XML format. haven't looked into this though.